src/Sortable/SortableListener.php

```diff
@@ -40,6 +40,9 @@ class SortableListener extends MappedEventSubscriber
     /** @var array<string, int> */
     private $maxPositions = [];
 
+    /** @var array<string, int> */
+    private $changeDueToMaxPosition = [];
```

Comment:
IMO, this name SHOULD be pluralized.

Reply:
@phansys thanks. I reverted this change. I was trying to fix an issue, where there are records in DB, but someone wants to update all positions with starting position = 1 instead of 0 (you get duplicated position coming from the listener then). but its too much effort, and decided to fix it elsewhere instead. Leaving fix for first issue here